test/dotc/tests.scala

```diff
@@ -230,6 +230,40 @@ class tests extends CompilerTest {
   @Test def tasty_dotc_printing = compileDir(dotcDir, "printing", testPickling)
   //@Test def tasty_dotc_reporting = compileDir(dotcDir, "reporting", testPickling)
   @Test def tasty_dotc_util = compileDir(dotcDir, "util", testPickling)
+  @Test def tasty_core_annotations = compileFile(coreDir, "Annotations", testPickling)
```

Comment:
instead of having a test per file, would you consider using `compileList` to test them all together?